src/components/table/footer/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { Component, h } from "@stencil/core"
+
+@Component({
+	tag: "smoothly-table-footer",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class TableFooter {
+	render() {
+		return (
+			<div>
```

Comment:
this div has styling display: content.

Is it really needed? If it is, what is it needed for? :eyes:

Reply:
This follows the same approach is the table row, cause I want to be able to pass down cells into it.